tests/client.spec.ts

```diff
@@ -417,6 +467,44 @@ describe("Client", () => {
     });
   });
   describe("test cases", () => {
+    it("should get one test case", async () => {
+      const getTestCaseTool = getTool(clientTools, "getTestCase");
+      jest.spyOn(api, "getTestCase").mockResolvedValue({
```

Comment:
can we get createMockTestCase etc. methods with partials? would make these tests much more readable

Reply:
also why is this a spy, i hope the api is mocked in all these tests? ;)